src/lib/zkapp.ts

```diff
@@ -1031,6 +1052,14 @@ async function deploy<S extends typeof SmartContract>(
   return tx.sign().toJSON();
 }
 
+function Account(address: PublicKey, tokenId?: Field) {
```

Comment:
convenient helper (suggested by @Trivo25) to inspect another account in a zkApp. somewhat dangerous because it adds an extra party which you can't easily reuse for something else